documentation/sphinx/source/api-ruby.rst

```diff
@@ -837,6 +837,13 @@ Transaction options
 
 .. _transact:
 
+Transaction misc functions
```

Comment:
This has also been added under a link that is intended to apply to the following section, and it probably should be moved above the options section too.

Reply:
Done. Also added `Int64Future` class documentation.